config/locales/views/home/en.yml

```diff
@@ -1,5 +1,5 @@
 en:
-  home:
+  pages:
```

Comment:
By convention this file would also now be named `config/locales/views/pages/en.yml`